src/ctree-app/ctree-landing.html

```diff
@@ -236,6 +239,74 @@
           bottom: 10%;
       }
 
+      /* Feedback cards styles */
+
+      .feedback-container {
+        width: 85%;
+        text-align: center;
+        justify-content: center;
+        background-color: #FFFFFF;
+        margin-top: 15%;
+        margin-bottom: 5%;
+        padding-top: 2%;
+        padding-bottom: 2%;
+        margin-left: auto;
+        margin-right: auto;
+        box-shadow: 2px 2px;
+
+      }
+      .feedback-cards {
+        padding-top: 5%;
+        padding-bottom: 5%;
+        display: flex;
+        flex-wrap: wrap;
+      }
+
```

Comment:
This is a VERY minor point, but it's probably best to be consistent with empty lines between entries.  I like to use empty lines to separate logical groups, although you've done a good job labeling groups already so you may not need empty lines between items.

Reply:
i appreciate you pointing this out, I will work of referencing these coding practices.